examples/custom-log-format/README.md

```diff
@@ -6,12 +6,9 @@ This example lets you set the log-format for NGINX using the configmap reosurce
 kind: ConfigMap
```

Comment:
few more notes about the example:

(1)the description has a typo in `reosurce`

(2) a few suggestions  and a correction about the variable descriptions:

In addition to the [built-in NGINX variables](https://nginx.org/en/docs/varindex.html), you can also use the variables that the Ingress Controller configures:

* `$resource_type` - The type of the Kubernetes resource, which handled the client request. Could be `ingress`, `virtualserver` or `virtualserverroute`.
* `$resource_name` - The name of the resource.
* `$resource_namespace` - The namespace the resource exists in.
* `$service` - The name of the service the client request was sent to.